tdrs-backend/tdpservice/email/templates/upcoming-submission-deadline.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %}
+{% block content %}
+
+<!-- Body copy -->
+<p>Hello {{ first_name }},</p>
+<p>This is a friendly reminder that your data files are due in 5 days. </p>
+<p>Please sign in to the <a href="https://tanfdata.acf.hhs.gov/" target="_blank">TANF Data Portal</a> 
+to upload and submit data files for <b>Fiscal Year {{ fiscal_year }} - {{ fiscal_quarter}}</b> by <b>{{ submission_deadline }}.</b></p>
+
+<!-- This link uses descriptive text to inform the user what will happen with the link is tapped. -->
+<!-- It also uses inline styles since some email clients won't render embedded styles from the head. -->
+<!-- <a href="" style="color: #336a90; text-decoration: underline;">Reset your password now</a> -->
+<p><a class="button" href="https://tdp-frontend-a11y.app.cloud.gov/" style="background-color:#336a90;border-radius:4px;color:#ffffff;display:inline-block;font-family:sans-serif;font-size:18px;font-weight:bold;line-height:60px;text-align:center;text-decoration:none;width: auto; padding-left: 24px; padding-right: 24px;-webkit-text-size-adjust:none;">Submit your data files</a></p>
```

Comment:
@jtimpe the url here should be dynamic to the relevant frontend app. currently hardcoded to a11y app. im also wondering if this button is needed, since the preceding sentence includes the prod URL? maybe that should be dynamic too? cc: @reitermb 

![Screenshot 2024-07-15 174503](https://github.com/user-attachments/assets/5e60eddd-5a87-46fe-a95d-6b1837f0cf87)

Reply:
thank you! apologies for the oversight - i've replaced the hardcoded links with the backend-supplied variable